pkgs/top-level/all-packages.nix

```diff
@@ -535,6 +535,8 @@ with pkgs;
 
   bonnie = callPackage ../tools/filesystems/bonnie { };
 
+  ct = callPackage ../development/tools/ct { };
```

Comment:
Maybe this should be renamed to `container-linux-config-transpiler` to match the project name. The binary should still named `ct`.